src/vfocus.js

```diff
@@ -300,3 +300,13 @@ VFocus.prototype.children = function(){
 VFocus.prototype.addChild = function(child){
   this.vNode.children.push(child);
 };
+
+VFocus.prototype.indexOf = function(focus){
```

Comment:
Sorry I wasn't being entirely clear there. I meant you'd have to change the calling code as well because of that. And yes, `index` would be a better name come think of it.

I'm a bit hesitant to add "low level" things like this `indexOf` to `VFocus`. Feels like it'd be better to access the raw objects or encapsulate that logic someplace else.